grammars/variable.grm

```diff
@@ -40,6 +40,9 @@ synizesis = Optimize[
 diaeresis = Optimize[
     CDRewrite[("w" : "u") <10000>,
               "k" | "g", "", sigma_star, 'ltr', 'opt'] @
+    # "aw" diphthongs preceding final m may lenite.
```

Comment:
Okay getting back to this. A few notes:

1. "dipthongs" should be "diphthongs", with two _h_s, strange as that looks.
2. What if this also did the nasalization, is there any harm doing it that way? E.g.:

    CDRewrite[("wm": "ũː") <10000>, "", "kwe"? i.EOW, sigma_star, 'ltr', 'opt'] @ ...

That better maps onto what we do [here](https://github.com/CUNY-CL/latin_scansion/blob/9a170ae3f12ae056fd56c19439deda312d1e8a05/grammars/pronounce.grm#L110). The comment then should probably say something slightly different too, since while I guess this is sort of like lenition I'd just say that nasalization variably bleeds glide formation.

# Word-final <au> nasalization variably bleeds glide formation.